deployment/ccip/changeset/solana/cs_ping_pong.go

```diff
@@ -0,0 +1,254 @@
+	return nil
+}
+func StartPingPongContractChangeset(
+	e deployment.Environment,
+	c StartPingPongConfig,
+) (deployment.ChangesetOutput, error) {
+	if err := c.Validate(e); err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("invalid StartPingPongConfig: %w", err)
+	}
+
+	s, err := ccipChangeset.LoadOnchainStateSolana(e)
+	if err != nil {
+		e.Logger.Errorw("Failed to load existing onchain state", "err", err)
+		return deployment.ChangesetOutput{}, err
+	}
+
+	chain := e.SolChains[c.ChainSelector]
+	chainState := s.SolChains[c.ChainSelector]
+
+	ppConfigPDA, _, err := solanaStateUtils.FindPingPongDemoConfigPDA(chainState.PingPong)
+	if err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("failed to find ping pong config pda: %w", err)
+	}
+
+	ppSendSignerPDA, _, err := solanaStateUtils.FindPingPongCCIPSendSignerPDA(chainState.PingPong)
+	if err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("failed to find ping pong config pda: %w", err)
+	}
+
+	destChainStatePDA, err := solanaStateUtils.FindDestChainStatePDA(c.ChainSelector, chainState.Router)
+	if err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("failed to find ping pong config pda: %w", err)
+	}
+
+	routerNoncePDA, err := solanaStateUtils.FindNoncePDA(c.ChainSelector, solana.PublicKey(*chain.DeployerKey), chainState.Router)
+	if err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("failed to find ping pong config pda: %w", err)
+	}
+
+	feeBillingSignerPDA, _, err := solanaStateUtils.FindFeeBillingSignerPDA(chainState.Router)
+	if err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("failed to find ping pong config pda: %w", err)
+	}
+
+	fqBillingTokenConfigPDA, _, err := solanaStateUtils.FindFqBillingTokenConfigPDA(nil, chainState.FeeQuoter)
+	if err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("failed to find ping pong config pda: %w", err)
+	}
+
+	rmnRemoteConfigPDA, _, err := solanaStateUtils.FindRMNRemoteConfigPDA(chainState.RMNRemote)
+	if err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("failed to find ping pong config pda: %w", err)
+	}
+
+	rmnRemoteCursesPDA, _, err := solanaStateUtils.FindRMNRemoteCursesPDA(chainState.RMNRemote)
+	if err != nil {
+		return deployment.ChangesetOutput{}, fmt.Errorf("failed to find ping pong config pda: %w", err)
+	}
+
+	initTx, err := solPingPong.NewStartPingPongInstruction(
+		ppConfigPDA,
+		chain.DeployerKey.PublicKey(),
+		ppSendSignerPDA,
+		nil, // feeTokenProgram ag_solanago.PublicKey,
+		nil, // feeTokenMint ag_solanago.PublicKey,
+		nil, // feeTokenAta ag_solanago.PublicKey,
+		chainState.Router,
+		chainState.RouterConfigPDA,
+		destChainStatePDA,
+		routerNoncePDA,
+		chainState.Receiver, // TODO is this right?
+		feeBillingSignerPDA,
+		chainState.FeeQuoter,
+		chainState.FeeQuoterConfigPDA,
+		nil, // feeQuoterDestChain ag_solanago.PublicKey,
+		fqBillingTokenConfigPDA,
+		nil, // feeQuoterLinkTokenConfig ag_solanago.PublicKey,
```

Comment:
This is the fee quoter billing config PDA for the link token. See [the rust code](https://github.com/smartcontractkit/chainlink-ccip/blob/d2fd3633f4854492d233a0b545dd17895cfcdc4e/chains/solana/contracts/programs/ccip-router/src/context.rs#L391-L399) and [go test code](https://github.com/smartcontractkit/chainlink-ccip/blob/main/chains/solana/contracts/tests/examples/ping_pong_test.go#L57)

Fee quoter always requires the LINK token config regardless of what token you're paying fees with. Having said that, when you're paying with LINK, then it's the same account passed twice as `fqBillingTokenConfigPDA` and here as the link token config.